packages/base/src/delegate/ResizeHandler.js

```diff
@@ -1,65 +1,63 @@
-import NativeResize from "./NativeResize.js";
-import CustomResize from "./CustomResize.js";
+import { nativeObserve, nativeUnobserve } from "./NativeResize.js";
+
+let observe = nativeObserve;
+let unobserve = nativeUnobserve;
+
+/**
+ * Set a callback to be executed whenever a DOM node needs to be observed for size change.
+ * @public
+ * @param callback
+ */
+const setResizeHandlerObserveCallback = callback => {
```

Comment:
Observe is a method on the ResizeObserver instance, not a callback, so something like `setResizeObserver`, `setResizeObserverFn`, `setResizeObserverMethod` or anything you suggest is shorter and more acurate.